_ont/ont-sercomm-fg1000b-11.md

```diff
@@ -44,14 +44,173 @@ The ONT seems only to display output of the ROM CFE and flash CFE, but don't all
 
 {% include serial_dump.html file="fg1000b-11_boot_cfe.txt" alt="Sercomm FG1000B.11 CFE boot dump" title="Sercomm FG1000B.11 CFE boot dump" %}
 
+## Root procedure
+[See how the enable telnet/ssh section](/ont-sercomm-fg1000b-11/#enable-telnetsshserial)
+
 ## List of software versions
 
 Current only version seen is: 090144.1.0.001
 
+## List of partitions
+`cat /proc/mtd`
+
+| dev:   | size     | erasesize | name                 |
+| ------ | -------- | --------- | -------------------- |
+| mtd0:  | 00200000 | 00020000  | "CfeROM              |
+| mtd1:  | 00400000 | 00020000  | "CfeRAM1             |
+| mtd2:  | 00400000 | 00020000  | "CfeRAM2             |
+| mtd3:  | 000a0000 | 00020000  | "FlashMAP            |
+| mtd4:  | 000a0000 | 00020000  | "SN                  |
+| mtd5:  | 00140000 | 00020000  | "Protect             |
+| mtd6:  | 01b80000 | 00020000  | "Rootfs1             |
+| mtd7:  | 00c80000 | 00020000  | "Lib1                |
+| mtd8:  | 01b80000 | 00020000  | "Rootfs2             |
+| mtd9:  | 00c80000 | 00020000  | "Lib2                |
+| mtd10: | 000a0000 | 00020000  | "Bootflg             |
+| mtd11: | 000a0000 | 00020000  | "Rootfs1_Info        |
+| mtd12: | 000a0000 | 00020000  | "Lib1_Info           |
+| mtd13: | 000a0000 | 00020000  | "Rootfs2_Info        |
+| mtd14: | 000a0000 | 00020000  | "Lib2_Info           |
+| mtd15: | 00280000 | 00020000  | "XMLConfig           |
+| mtd16: | 00280000 | 00020000  | "Erasable_XML_CFG    |
+| mtd17: | 00960000 | 00020000  | "AppData             |
+| mtd18: | 00140000 | 00020000  | "Yaffs               |
+| mtd19: | 010c0000 | 00020000  | "Reserve             |
+| mtd20: | 00930000 | 0001f000  | "rootfs_ubifs        |
+| mtd21: | 0029bf98 | 0001f000  | "filestruct_full.bin |
+| mtd22: | 003bd000 | 0001f000  | "lib_squashfs        |
+
+# Useful files and binaries
+
+{% include alert.html content="NanD MTD 5 mounted as  `/tmp/var_link_dir/ft` contains all serials and mac address of the ONT, please consider backup before any hack, files are: `customer_sn,gpon_sn,hw_version,mac_addr,pcba_sn`" alert="Warning" icon="svg-warning" color="red" %}
```

Comment:
Here and in other locations, it's better to use the original mount point, which is `/var/ft`.

The `/tmp/var_link` comes from the `/var -> /tmp/var_link_dir` symlink, all scripts and binaries use `/var` paths.